src/amo/components/AddonDetail.js

```diff
@@ -113,6 +114,31 @@ export class AddonDetailBase extends React.Component {
     );
   }
 
+  readReviewsFooter() {
+    const { addon, i18n } = this.props;
+    let content;
+
+    if (addon.ratings.count) {
+      const count = addon.ratings.count;
+      const linkText = i18n.sprintf(
+        // TODO: localize the review count? Or YAGNI?
+        ngettext('Read %(count)s review', 'Read %(count)s reviews', count),
+        { count },
+      );
+
+      content = (
+        <Link className="AddonDetail-all-reviews-link"
+          to={`/addon/${addon.slug}/reviews/`}>
+          {linkText}
+        </Link>
+      );
+    } else {
+      content = <span>{i18n.gettext('No reviews yet')}</span>;
+    }
+
+    return <div className="AddonDetail-read-reviews-footer">{content}</div>;
```

Comment:
You should be able to grab the footer with like `.AddonDetail-overall-rating .Card-footer`.  We don't have a great way of stating that a selector is "public" for a component but I doubt that one would change.